### ISSUE TITLE
Chipset ICN2037 and RT5957

### ISSUE THREAD

**Issue by pyzimmer** [pixelmatix/SmartMatrix#92]
Hello, I bought this panel  : https://fr.aliexpress.com/item/32836331716.html?spm=a2g0s.9042311.0.0.232f6c37cu1lnO

The chipset is : 
- ICN2037 : http://en.chiponeic.com/content/details36_245.html
and state graphs : https://www.mikrocontroller.net/attachment/392914/ICN2037.pdf
- RT5957 : http://www.dlfushi.com/uploads/201805/5b03e7bc77f3a.pdf
- 74HC245 chip : https://assets.nexperia.com/documents/data-sheet/74HC_HCT245.pdf

I am able to display only red lines in strange order from MODE_MAP_TESTING of MultiRowRefreshMapping.ino :
![20191114_201521](https://user-images.githubusercontent.com/43472705/68891013-571ba980-0720-11ea-9e4a-be306f424246.jpg)
vidéo : https://photos.app.goo.gl/WxDnsUxLfPkXDhWPA

I try with MODE_MAP_REVERSE_ENGINEERING, with 4 or 8 lines height, I get only red led flickering

My controller is ESP32.
I try with the library PxMatrix, I have similar issue.  

Please note that your library works fine with another model of LED panel.
Congratulation for your work !

**Comment by pyzimmer** [comment]
Do you think I have any chance to get it work by creating a new PanelMappingEntry ?

**Comment by embedded-creations** [comment]
No, I think the issue is something like a wiring issue, level shifters are required for your panel, or there is a chipset incompatibility.  Sorry, I don't have any clues of what to try first after watching the video.

**Comment by embedded-creations** [comment]
Make sure you're not trying to use the SmartLED Shield V4 mode if you're just directly connecting pins to the Teensy (without the flip flop circuit)

**Comment by pyzimmer** [comment]
Thank you. No, the SmartLED Shield V4 mode is commented.
The wiring works fine with another panel model. I try with level shifter : same result....

**Comment by pyzimmer** [comment]
hello I found a similar issue with the RT5957 chip : https://community.pixelmatix.com/t/led-module-p2-128-64-1-32-driver/414/4
